src/development/ui/advanced/actions_and_shortcuts.md

```diff
@@ -79,7 +79,7 @@ bindings, and the implementation of those bindings, are in different places.
 If indeed all that is needed is a callback, without all the complexity (or
 flexibility) of `Actions` and `Shortcuts`, you can already use a `Focus` widget
 for this. For example, here's the implementation of Flutter's simple
-[`CallbackShortcuts`][] widget (available on the dev branch) that takes a map of
+[`CallbackShortcuts`][] widget that takes a map of
```

Comment:
Please merge the following line onto this line to fill things out to 80 chars.

Reply:
Done!